test/e2e/states.go

```diff
@@ -135,3 +135,27 @@ func ifaceDownIPv4Disabled(iface string) nmstatev1alpha1.State {
         enabled: false
 `, iface))
 }
+
+func ifaceUpWithDynamicIP(infName string) nmstatev1alpha1.State {
+	return nmstatev1alpha1.NewState(fmt.Sprintf(`interfaces:
+    - name: %s
+      type: vlan
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+`, infName))
+}
+
+func ifaceDownDynamicIPDisabled(infName string) nmstatev1alpha1.State {
```

Comment:
We don't put he interface down we just flush ips.

Reply:
Unsure what you mean,
1. The mehod is called ifaceDownDynamicIPDisabled but the NNCP has state=up - fixed
If you that I shouldn't change the state to down - before applying the ifaceUpWithDynamicIP  the interface is down.
So I'd like to apply a an NNCP that will return the node network configuration back to it's original state.

Reply:
i guess I just need to use the vlanAbsent NNCP and the iface will go away along with the ips